modules/nw-http2-haproxy.adoc

```diff
@@ -0,0 +1,46 @@
+// Module included in the following assemblies:
+//
+// * networking/ingress-operator.adoc
+
+[id="nw-http2-haproxy{context}"]
+= HTTP/2 Ingress connectivity
```

Comment:
Is this more like _Enabling HTTP/2 Ingress connectivity_ or similar?

Reply:
I think so; if you add the annotation you're explicitly enabling HTTP/2.